piano-hero/core/src/main/java/com/novoda/pianohero/Note.java

```diff
@@ -0,0 +1,39 @@
+package com.novoda.pianohero;
+
+public class Note {
+
+    private static final int LOWEST_MIDI_NOTE = 0;
+    private static final int HIGHEST_MIDI_NOTE = 127;
+
+    private final int midiNoteNumber;
+
+    public Note(int midiNoteNumber) {
+        this.midiNoteNumber = checkNotOutOfRange(midiNoteNumber);
+    }
+
+    private int checkNotOutOfRange(int midiNoteNumber) {
```

Comment:
this is the full range for MIDI - since our app will be limited in scope (C4-B5) should this take into account that range?

Or should we just avoid adding Songs that use notes outside this range? The app would be flexible enough to handle anything in this range.

The limiting factors would be the inputs/Views for output. E.g. you couldn't represent C0 on a treble clef without a ridiculous number of lines and you can't even input C0 using even a full size piano (lowest note is A0).